modules/virt-creating-a-service-from-a-virtual-machine.adoc

```diff
@@ -82,8 +82,11 @@ spec:
         - name: cloudinitdisk
           cloudInitNoCloud:
             userData: |
-              #!/bin/bash
-              echo "fedora" | passwd fedora --stdin
+             #cloud-config
+             user: fedora
+             password: fedora
+             chpasswd: {expire: False}
+...
```

Comment:
"When truncating YAML, always precede an ellipsis with a pound so that the YAML is valid." The contribution guide states this, but its examples do not show it. 😫

Reply:
Wasn't aware of that requirement and I don't think I have seen the `#` sign being used consistently across the code examples in our docs.